### ISSUE TITLE
Gestures are not working in certain Mozilla pages

### ISSUE THREAD

**Issue by vishvega** [Robbendebiene/Gesturefy#449]
### Describe your matter in detail
I'm aware of listed Limitations / Problems, (_The addon does not work on addons.mozilla.org, pure SVG pages and internal pages_), 
I recently faced a problem at Mozilla Support page. Gestures aren't working. below listed the pages I've visited and didn't work on any of these pages.

https://support.mozilla.org/en-US/kb/rate-your-firefox-experience-heartbeat
https://support.mozilla.org/en-US/get-involved
https://support.mozilla.org/en-US/
https://support.mozilla.org/en-US/products/firefox

So, basically I think all the pages in support.mozilla.org is affected. I'm not sure it has some script running for disable dragging or some specific mouse events. but it has no restrictions on right click or common left click.

Page isn't purely SVG based, neither dynamically rendered. I tried gestures after the full page load. 

### Your System
- Operating system: Windows 10 and Kubuntu 19.10
- Firefox version: 73.0.1 64bit
- Gesturefy version: 2.1.6

**Comment by Robbendebiene** [comment]
The add-on description is incomplete, support.mozilla.org is also a privileged URL. So extension are not allowed to run there. You can find a full list somewhere here in the [Troubleshooting guide](https://github.com/Robbendebiene/Gesturefy/wiki/Troubleshooting)

Btw: you can allow these sites in about:config as mentioned here: https://github.com/Robbendebiene/Gesturefy/issues/6
